Header header
float64 azimuth
float64 elevation
float64 distance
int32 id
bool[3] trust
float64[3] standard_deviation